src/OrbitGl/ThreadStateBar.cpp

```diff
@@ -95,6 +96,18 @@ static Color GetThreadStateColor(ThreadStateSlice::ThreadState state) {
   }
 }
 
+[[nodiscard]] static std::string GetWakeupReason(ThreadStateSliceInfo::WakeupReason reason) {
+  switch (reason) {
+    case ThreadStateSliceInfo::WakeupReason::kNotApplicable:
+      return "not applicable";
```

Comment:
If this function is only called when `thread_state_slice->wakeup_reason() != ThreadStateSliceInfo::WakeupReason::kNotApplicable`, I would consider `ORBIT_UNREACHABLE();` here.